server/src/main/java/org/apache/iotdb/db/query/context/QueryContext.java

```diff
@@ -87,4 +90,15 @@ public long getJobId() {
     return jobId;
   }
 
+  public long getQueryTimeLowerBound() {
+    return queryTimeLowerBound;
+  }
+
+  public void setQueryTimeLowerBound(long queryTimeLowerBound) {
+    this.queryTimeLowerBound = queryTimeLowerBound;
+  }
+
+  public boolean chunkNotSatisfy(ChunkMetaData chunkMetaData) {
```

Comment:
do we need a pageNotSatisfy()?

Reply:
That is done by the time filter and have nothing to do with here.